saleor/graphql/attribute/filters.py

```diff
@@ -124,3 +138,129 @@ class Meta:
 class AttributeValueFilterInput(FilterInputObjectType):
     class Meta:
         filterset_class = AttributeValueFilter
+
+
+class AttributeInputTypeEnumFilterInput(graphene.InputObjectType):
+    eq = AttributeInputTypeEnum(description=FilterInputDescriptions.EQ, required=False)
+    one_of = NonNullList(
+        AttributeInputTypeEnum,
+        description=FilterInputDescriptions.ONE_OF,
+        required=False,
+    )
+
+
+class AttributeEntityTypeEnumFilterInput(graphene.InputObjectType):
+    eq = AttributeEntityTypeEnum(description=FilterInputDescriptions.EQ, required=False)
+    one_of = NonNullList(
+        AttributeEntityTypeEnum,
+        description=FilterInputDescriptions.ONE_OF,
+        required=False,
+    )
+
+
+class AttributeTypeEnumFilterInput(graphene.InputObjectType):
+    eq = AttributeTypeEnum(description=FilterInputDescriptions.EQ, required=False)
+    one_of = NonNullList(
+        AttributeTypeEnum,
+        description=FilterInputDescriptions.ONE_OF,
+        required=False,
+    )
+
+
+class MeasurementUnitsEnumFilterInput(graphene.InputObjectType):
+    eq = MeasurementUnitsEnum(description=FilterInputDescriptions.EQ, required=False)
+    one_of = NonNullList(
+        MeasurementUnitsEnum,
+        description=FilterInputDescriptions.ONE_OF,
+        required=False,
+    )
+
+
+def filter_attribute_name(qs, _, value):
+    return filter_by_string_field(qs, "name", value)
+
+
+def filter_attribute_slug(qs, _, value):
+    return filter_by_string_field(qs, "slug", value)
+
+
+def filter_with_choices(qs, _, value):
+    lookup = Q(input_type__in=AttributeInputType.TYPES_WITH_CHOICES)
+    if value is True:
+        return qs.filter(lookup)
+    elif value is False:
+        return qs.exclude(lookup)
+    return qs
+
+
+def filter_attribute_input_type(qs, _, value):
+    return filter_by_string_field(qs, "input_type", value)
+
+
+def filter_attribute_entity_type(qs, _, value):
+    return filter_by_string_field(qs, "entity_type", value)
+
+
+def filter_attribute_type(qs, _, value):
+    return filter_by_string_field(qs, "type", value)
+
+
+def filter_attribute_unit(qs, _, value):
+    return filter_by_string_field(qs, "unit", value)
+
+
+def filter_attribute_value_required(qs, _, value):
+    return qs
```

Comment:
Are we sure that we don't want to add any filters here? 
If yes we should add a comment here.